### PR TITLE
Refactor admin SCSS for better use of uikit

### PR DESCRIPTION
**Pull request by maxious**

### CODE REVIEW

File: app/assets/scss/_diff.scss [hunk 1]
```diff
@@ -1,15 +1,15 @@
 .diff {
-  margin-bottom: $gutter;
+  //margin-bottom: $gutter;
```
Comment (matt-sm) [inline app/assets/scss/_diff.scss]: delete rather than comment?

**Comment by matt-sm** [comment]
Is it worth getting @dominikwilkowski to review?

**Comment by maxious** [comment]
If he can approve, sure ;)

**Comment by dominikwilkowski** [comment]
Can I talk to someone of you about this? I'm interested how you digest the ui-kit in `package.json` and how you add it.

**Comment by dominikwilkowski** [comment]
@maxious just talked me through this and it seems a big improvement over integrating the ui-kit by hand. I would add what version of neat you are using as a comment in your files just to make sure you can update if we do though. Thanks @maxious :)

**Pull request merged**